main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        player_data = json.load(file)
+
+    for player_name, player_data in player_data.items():
```

Comment:
The variable `player_data` is being overwritten in the loop. Consider renaming the loop variable to avoid confusion, such as `player_info`.